modules/tinymce/src/core/test/ts/browser/keyboard/ArrowKeysEdgeCefTest.ts

```diff
@@ -0,0 +1,282 @@
+import { Keys } from '@ephox/agar';
+import { describe, it, context } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import * as CaretContainer from 'tinymce/core/caret/CaretContainer';
+
+describe('browser.tinymce.core.keyboard.ArrowKeysEdgeCefTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+  }, [], true);
+
+  const assertStartContainer = (editor: Editor, f: (node: Node) => boolean) => {
```

Comment:
Is the predicate option needed here from what I can tell it's always getting the same predicate passed in in this file.